src/main/java/bio/terra/service/snapshot/flight/create/CreateSnapshotPrimaryDataFullViewGcpStep.java

```diff
@@ -41,10 +39,8 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
      * from the dataset tables, we will need to get the table's live views
      */
     Instant createdAt = CommonFlightUtils.getCreatedAt(context);
-    SnapshotRequestContentsModel contentsModel = snapshotReq.getContents().get(0);
     Snapshot snapshot = snapshotDao.retrieveSnapshotByName(snapshotReq.getName());
-    Dataset dataset = datasetservice.retrieveByName(contentsModel.getDatasetName());
-    bigQuerySnapshotPdao.createSnapshotWithLiveViews(snapshot, dataset, createdAt);
```

Comment:
if the dataset name is now correctly set on the snapshotRequestModel, why do these steps need to change?

Reply:
In general, we still want to move to using ID and not refetching the dataset in scope all the time. We already have dataset, lets reuse it.